src/full_defines.jl

```diff
@@ -561,9 +198,12 @@ const CPX_PARAM_RANDOMSEED = convert(Cint,1124)
 const CPX_PARAM_DETTILIM = convert(Cint,1127)
 const CPX_PARAM_FILEENCODING = convert(Cint,1129)
 const CPX_PARAM_APIENCODING = convert(Cint,1130)
-const CPX_PARAM_SOLUTIONTARGET = convert(Cint,1131)
+const CPX_PARAM_OPTIMALITYTARGET = convert(Cint,1131)
 const CPX_PARAM_CLONELOG = convert(Cint,1132)
 const CPX_PARAM_TUNINGDETTILIM = convert(Cint,1139)
+const CPX_PARAM_CPUMASK = convert(Cint,1144)
+const CPX_PARAM_SOLUTIONTYPE = convert(Cint,1147)
+const CPX_PARAM_SOLUTIONTARGET = convert(Cint,1148)
```

Comment:
Unfortunate that this changed value from 12.6: http://www-01.ibm.com/support/knowledgecenter/#!/SSSA5P_12.6.0/ilog.odms.cplex.help/CPLEX/Parameters/topics/SolutionTarget.html

Will have to think about how (if) we want to support multiple versions of params at once (not that we do that now...)